File: reverseNumber/reverseNumber.java
package reverseNumber;
import java.util.*;
class ReverseNumber{
public static void main(String[] args){
        int a=12;
        int ans=reverseNumber(a);
        
        System.out.println(ans);
}
public static int reverseNumber(int n){
    int rv=0;
    while(n!=0){
        int dig=n%10;
        rv=rv*10+dig;
        n=n/10;
    }
    return rv;
}
}

File: reverseNumber/reverseNumber.js

// let a=12;
// let ans=reverseNumber(a);

// console.log(ans);
// function reverseNumber(a){
//     let temp=a;
//     let rv=0;
//     while(a!==0){
//         let dig=a%10;
//         rv=rv*10+dig;
//         a=Math.floor(a/10);
        
//     }
//     return rv;

// }
const reverseNumber=(n)=>{
    let rv=0;
     while(n!==0){
        let dig=n%10;
        rv=rv*10+dig;
        n=Math.floor(n/10);

     }
     return rv;

}
console.log(reverseNumber(1234));